ice40/chipdb.py

```diff
@@ -76,6 +76,7 @@
 constids["MAC16"] = constids["ICESTORM_DSP"]
 constids["HFOSC"] = constids["ICESTORM_HFOSC"]
 constids["LFOSC"] = constids["ICESTORM_LFOSC"]
+constids["SMCCLK"] = constids["ICESTORM_SMCCLK"]
```

Comment:
The ICESTORM_ prefix is added when we modify a primitive for any reason (such as adding the _FABRIC ports and trimming to oscillators, or converting a bus to a list of signals). I don't think it's needed for SMCCLK, which might as well stay as just SMCCLK.